.changeset/sweet-berries-begin.md

```diff
@@ -0,0 +1,44 @@
+---
+'@astrojs/starlight': minor
+---
+
+Adds Expressive Code as Starlight’s default code block renderer
+
+⚠️ **Potentially breaking change:**
+This addition changes how Markdown code blocks are rendered to use [Expressive Code](https://github.com/expressive-code/expressive-code/tree/main/packages/astro-expressive-code).
+If you were already customizing how code blocks are rendered, you may wish to disable Expressive Code to preserve previous behavior:
+
+```js
+starlight({
+  expressiveCode: false,
+})
+```
```

Comment:
```suggestion
If you were already customizing how code blocks are rendered and don't want to use the [features provided by Expressive Code](https://starlight.astro.build/guides/authoring-content/#expressive-code-features), you can preserve the previous behavior by setting the new config option `expressiveCode` to `false`.
```

Reading the current text gave me the impression that we're subtly recommending disabling Expressive Code ("you may wish to", including a prominent example of the code to disable it), so I tried to make this more neutral and a bit less prominent by converting the example block to inline text. Of course I'm biased though, so I'm happy to hear your thoughts on this. :)

Reply:
Hehe. Sure, your version works too. I think technically we ARE recommending this for people using some other custom code block handling, but making it smaller is fine by me.